code/datums/datacore.dm

```diff
@@ -281,8 +281,10 @@ GLOBAL_VAR_INIT(record_id_num, 1001)
 		G.fields["photo-south"] = "'data:image/png;base64,[icon2base64(icon(G.fields["photo"], dir = SOUTH))]'"
 		G.fields["photo-west"] = "'data:image/png;base64,[icon2base64(icon(G.fields["photo"], dir = WEST))]'"
 		if(H.gen_record && !jobban_isbanned(H, "Records"))
+			G.fields["record"] = H.gen_record
```

Comment:
Any reason you made another field for the exact same data?

Reply:
Oh, I added these in when I was debugging why my character didn't have any records - it turns out I forgot to set them up on my test server.. anyway, this field isn't needed so it'll be removed